packages/template-retail-react-app/app/pages/login/index.jsx

```diff
@@ -105,15 +134,27 @@ const Login = () => {
                 marginBottom={8}
                 borderRadius="base"
             >
-                <LoginForm
-                    form={form}
-                    submitForm={submitForm}
-                    clickCreateAccount={() => navigate('/registration')}
-                    handleForgotPasswordClick={() => navigate('/reset-password')}
-                    isPasswordlessEnabled={passwordless?.enabled}
-                    isSocialEnabled={social?.enabled}
-                    idps={social?.idps}
-                />
+                {!form.formState.isSubmitSuccessful && currentView === LOGIN_VIEW && (
+                    <LoginForm
+                        form={form}
+                        submitForm={submitForm}
+                        clickCreateAccount={() => navigate('/registration')}
+                        handlePasswordlessLoginClick={() => {
+                            setLoginType('passwordless')
+                        }}
+                        handleForgotPasswordClick={() => navigate('/reset-password')}
+                        isPasswordlessEnabled={isPasswordlessEnabled}
+                        isSocialEnabled={isSocialEnabled}
+                        idps={idps}
+                    />
+                )}
+                {form.formState.isSubmitSuccessful && currentView === EMAIL_VIEW && (
+                    <PasswordlessEmailConfirmation
+                        form={form}
+                        submitForm={submitForm}
+                        email={passwordlessLoginEmail}
+                    />
+                )}
```

Comment:
super nit but it looks like the email is rendered on the first line within the `/login` page while on the 2nd line on the modal
<img width="438" alt="Screenshot 2024-11-08 at 2 46 45 PM" src="https://github.com/user-attachments/assets/9ac09895-343b-461c-a9ea-3899d3eec690">
<img width="396" alt="Screenshot 2024-11-08 at 2 46 29 PM" src="https://github.com/user-attachments/assets/f9ef2502-7186-4953-902d-7da5d1a93329">